dev/Dockerfile.test

```diff
@@ -10,12 +10,18 @@ RUN apt-get update && \
     # cmake and protobuf-compiler required for onnx install
     cmake protobuf-compiler
 
+# Install miniforge
+RUN wget https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-x86_64.sh && \
+    bash Miniforge3-Linux-x86_64.sh -b -p /opt/conda && \
+    rm Miniforge3-Linux-x86_64.sh
+ENV PATH=/opt/conda/bin:$PATH
+
 # Install required python packages first
 ADD requirements /app/requirements
 RUN pip install -r requirements/test-requirements.txt -r requirements/lint-requirements.txt --no-cache-dir
 
 # Add the rest of files and install mlflow in editable form + packages necessary for development
 ADD . /app
-RUN pip install --no-cache-dir -e .
+RUN pip install --no-cache-dir -e .[pipelines]
```

Comment:
We removed the `[pipelines]` extra in `branch-2.0`, right?

Reply:
Thanks for the catch! Reverted this change.